CHANGES

```diff
@@ -8,6 +8,11 @@
  release branch.
 
  Changes between 1.1.0h and 1.1.1 [xx XXX xxxx]
+  *) Modified the random device based seed sources to keep the relevant
+     file descriptors open rather than reopening them on each access. This
```

Comment:
Maybe add this formulation as second sentence, stolen from RAND_add(3):

```
This allows such sources to operate in a chroot() jail without the
associated device nodes being available.
```

Reply:
Fair change.  I'll do it tomorrow morning (either 13½ hours if I get in before the meetings start or 18 hours if not).

Reply:
More than 18 hours have passed, so I'm thinking this as a gentle reminder ;-)

Reply:
Richard, see d12475d6b3d7e22e66fc9f14605aaeae7bbf76ec.

Reply:
I obviously need more coffee.  Sorry for the noise...